test/kino_db/connection_cell_test.exs

```diff
@@ -60,6 +60,34 @@ defmodule KinoDB.ConnectionCellTest do
                {:ok, db} = Kino.start_child({Exqlite, opts})\
                """
     end
+
+    test "restores source code from attrs with BigQuery" do
+      attrs = %{
+        "variable" => "db",
+        "type" => "bigquery",
+        "project_id" => "",
+        "credentials" => %{},
+        "default_dataset_id" => ""
+      }
+
+      {_kino, source} = start_smart_cell!(ConnectionCell, attrs)
+
+      assert source ==
+               """
+               scopes = ["https://www.googleapis.com/auth/cloud-platform"]
+               credentials = %{}
+
+               goth_opts = [
+                 name: Goth,
+                 http_client: &Req.request/1,
+                 source: {:service_account, credentials, scopes: scopes}
+               ]
+
+               opts = [goth: Goth, project_id: "", default_dataset_id: ""]
+               db = ReqBigQuery.attach(Req.new(), opts)
```

Comment:
this is fine but wouldn't the following be more idiomatic Req usage and thus what we should output?

```suggestion
               db =
                 Req.new()
                 |> ReqBigQuery.attach(goth: Goth, project_id: "", default_dataset_id: "")
                   
```